internal/helm/helm.go

```diff
@@ -0,0 +1,81 @@
+package helm
+
+import (
+	"os/exec"
+	"regexp"
+	"strings"
+
+	numaExec "github.com/numaproj-labs/numaplane/internal/util/exec"
+	"github.com/numaproj-labs/numaplane/pkg/apis/numaplane/v1alpha1"
+)
+
+var (
+	re = regexp.MustCompile(`([^\\]),`)
+)
```

Comment:
Is this PR up to date with the latest main branch? I don't think this helm change should be showing up here, right?

Reply:
I did a pull with main and there was a conflict, this might have gotten mixed up in this. I can revert that to whatever is actually on the main branch.

Reply:
I see now this folder wasn't there before, not sure how that got added. Will remove.